api/gloo/gloo/external/envoy/extensions/transformation_ee/transformation.proto

```diff
@@ -53,7 +53,6 @@ message Transformation {
     // Template is in the transformed request language domain
     oneof transformation_type {
         DlpTransformation dlp_transformation = 1;
-        AiTransformation ai_transformation = 2;
```

Comment:
I'm not understanding where this is coming from

Reply:
@andy-fong this was added as part of https://github.com/solo-io/solo-apis/pull/1442/files do you still need these apis in solo-apis?